content/applications/inventory_and_mrp/inventory/product_management/product_tracking/package.rst

```diff
@@ -48,147 +44,85 @@ the :guilabel:`Operations` heading, activate the :guilabel:`Packages` feature. T
    :align: center
    :alt: Activate the *Packages* setting in Inventory > Configuration > Settings.
 
-.. _inventory/warehouses_storage/package-use:
-
-Package use
-===========
-
-.. note::
-   The following section is **only** necessary when packages are intended to be used for cluster
-   pickings.
-
-.. seealso::
-   :doc:`Cluster picking <../../warehouses_storage/advanced_operations_warehouse/cluster_picking>`
-
-First, navigate to :menuselection:`Inventory app --> Configuration --> Settings`, and activate the
-:guilabel:`Batch Transfers` feature, located in the :guilabel:`Operations` section. Doing so makes
-the *Package Use* field appear on a package form.
+.. _inventory/warehouses_storage/pack:
 
-.. image:: package/enable-batch.png
-   :align: center
-   :alt: Activate the *Batch Transfers* feature in Inventory > Configuration > Settings.
+Pack items
+==========
 
-Then, go to the package form to configure the package intended for batch picking. To do that,
-navigate to :menuselection:`Inventory app --> Products --> Packages`.
+Products can be added to packages in any transfer by:
 
-On this page, add new packages by clicking :guilabel:`New`, or select an existing package. Doing so
-opens the package form, which contains the following fields:
+#. Clicking each *Detailed Operations* icon on the product line.
+#. Using the *Put in Pack* button to place everything in the transfer into a package.
 
-- :guilabel:`Package Reference` (required): name of the package.
-- :guilabel:`Package Type`: used for :ref:`configuring shipping boxes to ship to the customer
-  <inventory/warehouses_storage/package-type>`.
+Detailed operations
+-------------------
 
-  .. note::
-     :guilabel:`Package Type` is unnecessary for configuring batch transfer packages.
+On any transfer, add a product to a package by clicking the :guilabel:`⦙≣ (bulleted list)` icon in
```

Comment:
might be beneficial to link to another doc that explains what transfers are/how to navigate to them

Reply:
Further defined what a transfer is in the doc itself. We currently don't have a doc dedicated to transfers 

```
On any warehouse transfer (e.g. receipt, delivery order), add a product to a package by clicking the
:guilabel:`⦙≣ (bulleted list)` icon in the :guilabel:`Operations` tab.
```